site/src/component/ChangelogModal/ChangelogModal.tsx

```diff
@@ -34,7 +35,10 @@ const ChangelogModal = () => {
       </Modal.Header>
 
       <p className="modal-body">{DESCRIPTION}</p>
-      <img className="modal-img" src={changelogImage} alt="Screenshot or gif of new changes" />
+      <div className="modal-side-by-side">
```

Comment:
Not sure how I feel about there being a new class just for this change log – we should at least leave a comment saying to remove the class if we remove this element from the change log. Also, this will be the first time I concede that Tailwind is better for something.